src/platform/silabs/wifi/SiWx/WifiInterface.cpp

```diff
@@ -408,6 +411,44 @@ sl_status_t SetWifiConfigurations()
     return status;
 }
 
+/**
+ * @brief Callback function for the SL_WIFI_JOIN_EVENTS group
+ *
+ * This callback handler will be invoked when any event within join event group occurs, providing the event details and any associated data
+ * The callback doesn't get called when we join a network using the sl net APIs
+ *
```

Comment:
```suggestion
 *
  * @note In case of failure, the 'result' parameter will be of type sl_status_t, and the 'resultLenght' parameter should be ignored
  *
```